src/ra_machine.erl

```diff
@@ -230,6 +232,10 @@
 
 -callback tick(TimeMs :: milliseconds(), state()) -> effects().
 
+-callback eval_members(ra_server_id(),
+                       [ra_server_id()],
+                       MacState :: state()) -> effects().
```

Comment:
@kjnilsson Should we return list of effects, i.e `effects()`, or limit it to one effect, i.e `effect()`?